src/sentry/static/sentry/app/views/organizationDiscover/discover.jsx

```diff
@@ -370,8 +405,8 @@ export default class OrganizationDiscover extends React.Component {
           showEnvironmentSelector={false}
           onChangeProjects={this.updateProjects}
           onUpdateProjects={this.runQuery}
-          onChangeTime={this.updateDateTime}
-          onUpdateTime={this.runQuery}
+          onChangeTime={this.changeTime}
```

Comment:
We probably don't even need this, but just leaving it here in case.

Reply:
Is there any case in which we want to expose `onChangeTime`? When does it get triggered exactly?

Reply:
In that case we can also drop onChangeProjects

Reply:
Let's follow up with a PR on this because it would require MultiProject + Environment refactors. We'll condense the callbacks to just `onChange*` instead of having 2 separate steps, where `change` event happens when the dropdown is closed.

Reply:
👍